mbedtls.nimble

```diff
@@ -7,17 +7,11 @@
 # those terms.
 
 packageName   = "mbedtls"
-version       = "0.0.1"
+version       = "0.0.2"
 author        = "Status Research & Development GmbH"
 description   = "Mbed-TLS wrapper in Nim"
 license       = "MIT or Apache License 2.0"
 
 before install:
   exec "git submodule update --init"
```

Comment:
` exec "git submodule update --init"` 
is this needed?

Reply:
Hm. good question. I'll check if nimble does take into account submodules when installing dependencies. I genuinely have no idea.

Reply:
Just confirmed that it is necessary by doing this in https://github.com/status-im/nim-mbedtls/pull/7/commits/9f6c6d8e03b4e67133120788c6b0550a3482e3c9

```
git clone git@github.com:status-im/nim-mbedtls.git
cd nim-mbedtls
git checkout no-more-python 
nimble install
nimble c mbedtls.nim
```

```
/home/r/vacp2p/test/nim-mbedtls/mbedtls/platform_time.nim(4, 10) Error: cannot find: ./mbedtls/csources/library/platform_util.c
```

Reply:
this means we will still have dependency to `git`, but that is much smaller issue.

Reply:
Indeed! that one should be fine. According to https://nim-lang.github.io/nimble/index.html#system-requirements
> Nimble has some runtime dependencies on external tools, these tools are used to download Nimble packages. For instance, if a package is hosted on [GitHub](https://github.com/), you need to have [git](https://www.git-scm.com/) installed and added to your environment PATH